src/zoid/buttons/util.js

```diff
@@ -135,6 +135,12 @@ export function getNoPaylaterExperiment(fundingSource : ?$Values<typeof FUNDING>
     };
 }
 
+export function getVenmoAppLabelExperiment() : EligibilityExperiment  {
+    return {
+        enableVenmoAppLabel: false
```

Comment:
Hardcoded blocker so we don't forget about it

Reply:
Steve, I forgot to comment on this, the intention is to display the "App" label during the final rendering of the component, with that in mind the `experiment.enableVenmoAppLabel` that counts is the one coming from scnw -> pspb and then forwarded to the zoid component. Let me know if the implementation is correct in that sense or if we should add a new prop for this experiment (for your comments on doing the `getExperiments` thing I'm assuming we want to keep it as one prop)

Reply:
This is the desired behaviour 
https://user-images.githubusercontent.com/89796119/142225790-ebe5260a-b2a2-4ba4-87dd-46bba3535a06.mov

Reply:
It looks like the `App` suffix should be positioned lower.

Reply:
> 

Setting this false to indicate that it occurs on second render is not clear here.  Can we not make a decision in it's prerender to ignore?